src/scss/init/_variables.scss

```diff
@@ -61,6 +61,22 @@ $grid-breakpoints: (
   xxl: $desk,
 ) !default;
 
-
 // Set the grid sizes with a default value that can be overwritten on a per theme basis.
 $max-grid-width: 1024px !default;
+
```

Comment:
being fussy - don't declare these in the _variables file for now, just declare them at the top of the src/scss/3_components/navigation/_breadcrumb.scss file. Unlikely that they will need overridden, however the point to making them variables is that they can be changed more easily (if needed). Also add !default to the end of $breadcrumb-margin and $breadcrumb-border-bottom variables - this ensures that the value can be overridden.